source/_patterns/00-base/demo/type.twig

```diff
@@ -42,16 +42,72 @@
   {% endfor %}
 </ul>
 
-<p>To add to these items, use Sass variables that start with $fs-- in
+  <p>To add to these items, use Sass variables that start with $fs-- in
   _fonts.scss</p>
+<hr>
 
 <h1>Headings</h1>
-<h1 contenteditable>h1 {{ headline.short }}</h1>
-<h2 contenteditable>h2 {{ headline.short }}</h2>
-<h3 contenteditable>h3 {{ headline.short }}</h3>
-<h4 contenteditable>h4 {{ headline.short }}</h4>
-<h5 contenteditable>h5 {{ headline.short }}</h5>
-<h6 contenteditable>h6 {{ headline.short }}</h6>
+{#
+  These are heading styles, including both element and classed version.
+  Use the element version in markup following Accessibility guidelines.
+  Use the class version to match font styling of heading.
+  See https://webaim.org/techniques/semanticstructure/
+#}
+  <h1 contenteditable>h1 {{ headline.short }}</h1>
+  <h2 contenteditable>h2 {{ headline.short }}</h2>
+  <h3 contenteditable>h3 {{ headline.short }}</h3>
+  <h4 contenteditable>h4 {{ headline.short }}</h4>
+  <h5 contenteditable>h5 {{ headline.short }}</h5>
+  <h6 contenteditable>h6 {{ headline.short }}</h6>
+
+  <p class="my-4">You can use these as either the element or as a class. For example:</p>
+  <pre class="bg-light">
+    <code>
```

Comment:
Go ahead and remove leading whitespace that `<pre>` is trying to save. (Yes it will make markup look gross, but thus is `<pre>`). Do this for all.